diagnostic.md

```diff
@@ -3,36 +3,54 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+- Sets up a front end saving the developer from a lot of coding
+- has structured folder layout and file naming conventions making it better for
+  leaning.
+- aids in making front end reliable from earlier on in development process
```

Comment:
I'd say these are more advantages of front-end frameworks in general. Here are some things that Ember helps with out of the box:
Front end routing
Easily display dynamic content using two-way binding
No more AJAX - Ember takes care of making API calls for you.

.5/1